app/src/main/java/com/example/trempel/PdpFragment.kt

```diff
@@ -29,10 +31,13 @@ internal class PdpFragment : Fragment() {
             this.lifecycleOwner = this@PdpFragment
             this.viewModel = this@PdpFragment.viewModel
         }.also {
-            viewModel.loadProduct()
+            observeLoadProduct()
```

Comment:
Looks like a redundant change. I don't think that we should wrap load call in `observe` prefixed function since we're not observing it anyways

Reply:
done